### PR TITLE
Updated _data files for 3_8_0

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [* ] No 

# Changes

### CODE REVIEW

File: _data/es/navigation.yml [hunk 1]
```diff
@@ -1,17 +1,15 @@
 nav:
 - title:                 Cómo usar Jamulus
   subfolderitems:
-    - page:              Cómo Empezar
-      url:               /wiki/Getting-Started
+    - page:              Configuración
+      url:               /wiki/Setup
```
Comment (ann0see) [inline _data/es/navigation.yml:5]: Not sure, but I think @gilgongo would like to change it back to Getting-Started. But I think he'll do it close to the release.

**Review by ann0see — APPROVED** [review]
Not yet merging since there's still one open question:

File: _data/es/navigation.yml (hunk 1 above)
Comment (gilgongo) [reply _data/es/navigation.yml:5]: Yes, I was going to just find/replace all the links to /wiki/Setup to /wiki/Getting-Started once I've moved the Setup content into that file so we can then remove *-Setup.md and have one less redirect.

Comment (ann0see) [reply _data/es/navigation.yml:5]: Ok. So this means that we can just merge this as is

**Pull request merged**